tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php

```diff
@@ -398,7 +399,9 @@ public function testSucessfulCase()
 							'payment_method' => [
 								'method' => 'id',
 								'id' => $subscription->paymentMethod()->id(),
-							]
+							],
+                            'amount' => $subscription->amount(),
```

Comment:
indentation

Reply:
Fixed... Tabs vs 4 spaces. IDE uses 4 spaces, couldn't see the difference in my IDE.